backend/dataall/modules/dataset_sharing/services/share_processors/lakeformation_process_share.py

```diff
@@ -155,6 +162,18 @@ def process_revoked_shares(self) -> bool:
             '##### Starting Revoking tables #######'
         )
         success = True
+
+        try:
+            if not self.check_catalog_account_exists_and_update_processor():
```

Comment:
`check_catalog_account_exists_and_update_processor` will return True even when `catalog_dict` = `None`. Which means that this `if` will always be True and lines 168,169 will never be executed (I think)

Reply:
Hi @dlpzx , Thanks for raising this. Yes, the if condition will never be executed. I had added this when the outer try catch was not present( See line 166 , 170 ) . I will correct it and send the updated PR